gtfu/src/test/java/gtfu/test/LoadAgencyDataTest.java

```diff
@@ -38,12 +41,27 @@ public LoadAgencyDataTest(String cacheDir, String agencyID) throws Exception {
 
     private static void usage() {
         System.err.println("usage: LoadAgencyDataTest -c|--cache-dir <cache-dir> -a|--agency-id <agency-id>");
+        System.err.println("usage: LoadAgencyDataTest -c|--cache-dir <cache-dir> -u|--url <url>");
+        System.err.println("    <url> is assumed to point to a plain text document that has an agency ID per line");
```

Comment:
Do we have a way to automatically generate the URL/text file, or is this another agency onboarding step?

Reply:
I think for now we want to manually curate it, so that new agencies are added but also so that dusty agencies get removed. Feel free to create something clever in the future that automates this, but for now I think the approach should be: when onboarding a new agency, copy file from bucket, add agency ID and look for old agencies to remove, then copy back to bucket